src/ape_ethereum/ecosystem.py

```diff
@@ -236,10 +236,25 @@ def encode_calldata(self, abi: ABI, *args) -> bytes:
         else:
             return HexBytes(b"")
 
-    def decode_calldata(self, abi: ABI, raw_data: bytes) -> Any:
+    def decode_calldata(self, abi: ABI, raw_data: bytes) -> Tuple[Any, ...]:
         output_types = [o.canonical_type for o in abi.outputs]
         try:
-            return abi_decode(output_types, raw_data)
+            vm_return_values = abi_decode(output_types, raw_data)
+            if not vm_return_values:
+                return vm_return_values
+
+            if not isinstance(vm_return_values, (tuple, list)):
```

Comment:
From what I could tell, the return values here were always a tuple anyway, but I added this part just to not make any assumptions about it